v2.1/admin-ui-statements-page.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Statements Page
+toc: false
+---
+
+The **Statements** page helps you identify the frequently executed or high latency [SQL statements](sql-statements.html). The **Statements** page also allows you to drill down to the details of an individual SQL statement by clicking on the statement to view the **Statement Details** page.
+
+To view the **Statements** page, open [http://localhost:8080/#/statements](http://localhost:8080/#/statements) in your browser (replacing `localhost` and `8080` with your node's host and port).
+
+<div id="toc"></div>
+
+{{site.data.alerts.callout_danger}}
+**This feature is a work in progress**. It will change leading up to the v2.1 release.
+{{site.data.alerts.end}}
+
+## Limitations
+
+- If you have multiple applications running on the cluster, the **Statements** page shows cumulative parameter values across all applications, while the **Statements Details** page shows the values for the first application only.
```

Comment:
It might be useful to link to documentation about setting the application, but as far as I can tell it's not really documented on it's own, just briefly on the cluster setting itself and then randomly mentioned elsewhere.